Detectors/Calibration/include/DetectorsCalibration/TimeSlot.h

```diff
@@ -34,13 +34,17 @@ class TimeSlot
 {
  public:
   TimeSlot() = default;
-  TimeSlot(TFType tfS, TFType tfE) : mTFStart(tfS), mTFEnd(tfE) {}
+  TimeSlot(TFType tfS, TFType tfE) : mTFStart(tfS), mTFEnd(tfE)
+  {
+    mTFStartMS = getStartTimeMS();
```

Comment:
This cannot work: getStartTimeMS() uses mRunStartOrbit which is set by the TimeSlotCalibration once the slot is created. Perhaps you can modify setRunStartOrbit method to also assign mTFStartMS whenever the mRunStartOrbit is set.

Reply:
Hi @shahor02 ,
Yes, I noticed. When I opened the PR, we were still testing, I forgot to add [WIP]. I will do so now. 
Thanks, 
Chiara